sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/AvoidExcessiveClassCoupling.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.IO;
+
+namespace Tests.Diagnostics
+{
+    interface IFoo { }
+    class FooBase : IFoo { }
+    class Foo1 : FooBase { }
+    public struct MyStruct { }
+
+    public abstract class TestCases // Noncompliant {{Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 7 to the maximum authorized 1 or less.}}
+//                        ^^^^^^^^^
+    {
+        // ================================================================================
```

Comment:
It was a pain to read the different parts when I was doing my tests. I could actually remove the sections now if you guys want.